docs/rules/valid-it-test-desc.md

```diff
@@ -0,0 +1,31 @@
+# Check "it()" and "test()" descriptions against a pre-configured regular expression
+
+This rule enforces the "it()" and "test()" descriptions to follow the desired format. 
```

Comment:
@lo1tuma good idea, let's make a more generic rule title and a configurable list of aliases. How about something like `valid-test-description` and, for a similar rule for `describe` - `valid-suite-description`?.. Thanks.